src/components/Button/Button.tsx

```diff
@@ -0,0 +1,17 @@
+import cn from "classnames";
+
+import "./button.scss";
+
+export default function Button({ label, color, onClick, variant }:any) {
```

Comment:
cool, now let's add some type in here:

1. In order to define a type for the props received (Props) you will replace `any` for the type that you want to, by default I use `Props` (type name is always in CapitalizeCase)

2. To define the type you use the keyword `type` instead `const`, like so:
```ts
type Props = {

}
```

3. Now how you use the props that you're receiving you'll be able to determine how `Type` to use:
`color` is a **string**, `variant` is a **string**, `label` is a **string**, and`onClick` is a **function**.

So our type will look like:
```ts
type Props = {
  color: string;
  label: string;
  variant: string;
  onClick: () => void
}
```

4. Finally you'll be able to attach is to your component props.
```suggestion
export default function Button({ label, color, onClick, variant }: Props) {
```